panel/package-lock.json

```diff
@@ -10936,6 +11043,18 @@
           "integrity": "sha1-M7QHd3VMZDJXPBIMw4CLvRDUfwQ=",
           "dev": true
         },
+        "vue": {
+          "version": "3.0.11",
```

Comment:
we're upgrading to Vue 3?

Reply:
How is that even possible? I ran an npm audit fix today. But this should never bump us to v3. How is it even possible that the Panel is still working like that.

Reply:
For some reason this is only in the package-lock.